snapcraft/plugins/make.py

```diff
@@ -94,9 +94,7 @@ def __init__(self, name, options, project):
         super().__init__(name, options, project)
         self.build_packages.append('make')
 
-    def build(self):
-        super().build()
-
+    def _build(self, env):
```

Comment:
If this is meant to be called by subclasses, I suggest you make it part of the plugin's public API.